modules/cert-manager-enable-metrics.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-operator-monitoring.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-enable-metrics_{context}"]
+= Enabling monitoring for the {cert-manager-operator}
+
+You can enable monitoring and metrics collection for the {cert-manager-operator} by using a service monitor to performing custom metrics scraping.
+
+.Prerequisites
+
+* You have access to the cluster with `cluster-admin` privileges.
+* The {cert-manager-operator} is installed.
+
+.Procedure
+
+. TODO: Step on verifying Prometheus selectors.
```

Comment:
Did you mean about the matchLabels field in the ServiceMonitor object or something else that I might be missing.

Reply:
Yes. Do we need the step for it?

Reply:
We could skip it for now.